cmds/umount/umount_linux.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
Makes absolute sense.
The same fix should be applied to: https://github.com/u-root/u-root/tree/master/uroot